compose/config/config.py

```diff
@@ -485,7 +485,6 @@ def resolve_volume_paths(service_dict, working_dir=None):
 
 def resolve_volume_path(volume, working_dir, service_name):
     container_path, host_path = split_path_mapping(volume)
-    container_path = os.path.expanduser(container_path)
```

Comment:
Because `container_path` should never be relative, so we don't need to call `expanduser` on it. @aanand spotted it last time when I was working through understanding the paths situation.